api/routes.go

```diff
@@ -55,6 +57,12 @@ func (r *RouteHandler) ConfigureRoutes(router *gin.Engine) {
 	}
 }
 
+// EnableAuth enables authentication middleware
+func (r *RouteHandler) EnableAuth(router *gin.Engine) {
+	signingKey := viper.GetString("tokensigningkey")
+	router.Use(auth.JWTAuth(auth.NewVaultTokenStore(""), signingKey, nil))
```

Comment:
The role should be an identifier which identifies the application, for example: `"telescopes"` later on we will configure Vault with that role.

Reply:
thanks @bonifaido, fxed